### PR TITLE
213 improve the introduction landing page

### PR DESCRIPTION
**Pull request by vordimous**
live version: https://vordimous.github.io/zilla-docs/next/

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -1,101 +1,127 @@
...
 
-## Getting Started
+Zilla is stateless, cloud-native, and supports various network and application protocols, including HTTP, Kafka, SSE, MQTT, gRPC, and WebSocket (additional protocols are on the way).
 
-Zilla can validate API requests and message streams using your OpenAPI and AsyncAPI schemas. Zilla implements the OpenAPI and AsyncAPI schemas directly meaning there are little or no changes necessary to start serving you existing operations through Zilla. You can just drop in an existing OpenAPI/AsyncAPI specification and Zilla can seamlessly integrate with your current API management workflows and tooling.
+![](/zilla-overview.png)
 
-You can see a working [Petstore Demo](https://github.com/aklivity/zilla-demos/tree/main/petstore) using OpenAPI/AsyncAPI schemas. Zilla can also define and proxy MQTT endpoints using a pair of AsyncAPI schemas. Check out the [Taxi Demo](https://github.com/aklivity/zilla-demos/tree/main/taxi) to see a Zilla MQTT proxy defined using AsyncAPI, which is deployed [Live](https://taxi.aklivity.io/) using Kubernetes.
+::: info Just want to build?
+Jump to the [quick start](./how-tos/quickstart.md) guide.
```
Comment (jfallows) [inline src/README.md:15]:
```suggestion
Jump to the [quickstart](./how-tos/quickstart.md) guide.
```

We use `quickstart` as a single word throughout.

File: src/README.md [hunk 2]
```diff
@@ -1,101 +1,127 @@
...
-```
+### HTTP/2, gRPC, SSE, MQTT, and Kafka Support + Protocol Mediation
 
-Specify your own `zilla.yaml` file.
+Inside Zilla, every protocol, whether it is TCP, TLS, HTTP, Kafka, gRPC, etc., is treated as a stream, so mediating between protocols simplifies to mapping protocol-specific metadata.
 
-```bash:no-line-numbers
-docker run -v ./zilla.yaml:/etc/zilla/zilla.yaml ghcr.io/aklivity/zilla:latest start -v
-```
+### Protobuf, Avro, OpenAPI, and AsyncAPI Support
```
Comment (jfallows) [inline src/README.md:34]: Do these belong together?

I realize there is implementation overlap in that Apicurio can host all the different types of artifacts for JSON schema, Protobuf schema, Avro schema, as well as OpenAPI specs and AsyncAPI specs, but IMHO they feel sufficiently different to highlight separately.

Also, we should highlight JSON schemas here too.

Comment (vordimous) [reply src/README.md:34]: Optional change:

```

### Protobuf, Avro, and JSON Schema payloads

Zilla supports payload schema specifications for message validation and translation.

### OpenAPI and AsyncAPI Support

Zilla supports different API schema specifications for message validation and API creation.
```

Comment (llukyanov) [reply src/README.md:34]: I'm fine with splitting these as per AJ's proposal.

Comment (vordimous) [reply src/README.md]: @jfallows pushed, let me know if that looks good.

Comment (jfallows) [reply src/README.md]: LGTM2. 👍

**Review by jfallows — APPROVED** [review]

**Pull request merged**